### PR TITLE
tag added for SSJ Playwright scenario

### PR DESCRIPTION
**Pull request by alenaminsk**

### CODE REVIEW

**Review by matakevin — APPROVED** [review]
Very quick and small PR, LGTM - thank you!

**Review by puzikovam2 — APPROVED** [review]
approved by puzikovam2

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature
Comment (iamdez99) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:1]: Please run this scenario again and make sure it passes, add screenshots wherever screenshots need to be added, if all is good please add the @Regression tag

**Review by iamdez99 — CHANGES_REQUESTED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature
Comment (alenaminsk) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:1]: done

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java
Comment (iamdez99) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java:1]: Cucumber steps should only have one line of code, you will have to add screenshots in the steps implementation classes.

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature
Comment (iamdez99) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/Applicant Profile.feature:1]: Sorry - I meant @Regression not @regression

**Review by iamdez99 — CHANGES_REQUESTED** [review]

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Pull request merged**